tests/Builder/Expression/ToDoubleOperatorTest.php

```diff
@@ -0,0 +1,33 @@
+<?php
+
+declare(strict_types=1);
+
+namespace MongoDB\Tests\Builder\Expression;
+
+use MongoDB\Builder\Expression;
+use MongoDB\Builder\Pipeline;
+use MongoDB\Builder\Stage;
+use MongoDB\Tests\Builder\PipelineTestCase;
+
+/**
+ * Test $toDouble expression
+ */
+class ToDoubleOperatorTest extends PipelineTestCase
```

Comment:
This is outside the scope of the PR, but why does the parameter type for `ToDoubleOperator::__construct()` include `non-empty-string` instead of just `string`? In `toDouble.yaml`, the `$expression` argument has a type `expression`.

I traced this to a "strings cannot be empty" comment in `OperatorGenerator::getAcceptedTypes()`, but it's unclear to me why that is. That's also the case for StrLenBytesOperator, which I'd expect would be able to take a string of any length.

Reply:
Out of zeal, I've put this type for all the strings, but that's probably overkill.

Reply:
I feel like there would be at least _some_ cases where an empty string is a legitimate argument for some operators. If you agree, consider opening a separate issue to investigate that at a later point.

Reply:
Tracked in [PHPLIB-1373](https://jira.mongodb.org/browse/PHPLIB-1373)